### ISSUE TITLE
[Bug]: trakt tv login doesn't work

### ISSUE THREAD

**Issue by frytex2** [nova-video-player/aos-AVP#1283]
### Problem description

the issue is about authorising nova to use trakt.tv after upgrade to the newest version.  

on the tv where I had a previous version, nova remains logged in and synces the data properly with trakt. 
on another tv set, where i removed the old version and installed the new one from scratch (so the settings were not inherited), authentication to trakt from within the nova player opens the trakt login page, and when i log in and get the Authorise yes/no question, when i click "yes" i get the "400 bad request error". 
being curious like a cat i logged off from trakt on the tv where the settings were inherited (and which worked)  and now i cannot authorize nova to use trakt on both tv's :/
in some other github thread i found this, regarding the error 400: "in your app, are you using the trakt.tv or the api.trakt.tv domain? it should be main trakt.tv domain for all authentication UI". 
so maybe this is the issue

### Steps to reproduce the issue

run nova, log out from trakt (if already logged in from previous nova version), log back in trakt. 

### Your phone/tablet/androidTV model

sony bravia

### Operating system version

no idea. newest. :) 

### Application version and app store

6.2.9.7

**Comment by courville** [comment]
We are using [api.trakt.tv](http://api.trakt.tv/) according to https://trakt.docs.apiary.io/#introduction/status-codes

Where did you get that we need to use trakt.tv?

I retested with email login/password and it works on my side with nova (not saying that it is not a problem for others: I am unable to reproduce).

**Comment by kris3713** [comment]
> We are using [api.trakt.tv](http://api.trakt.tv/) according to https://trakt.docs.apiary.io/#introduction/status-codes
> 
> Where did you get that we need to use trakt.tv?
> 
> I retested with email login/password and it works on my side with nova (not saying that it is not a problem for others: I am unable to reproduce).

I have video proof of this happening to me as well.

https://github.com/user-attachments/assets/3ece1d02-3020-4abf-a76e-bef8aa3dd4c2

EDIT: I am no longer using Trakt. I will ignore any replies (including mentions) in this thread unless it is important.

**Comment by courville** [comment]
I am not disputing the fact that it happens to you but unfortunately, I cannot reproduce on my side.

**Comment by frytex2** [comment]
yes this is exactly my error. I get it on Android TV on Sony Braviaa

śr., 23 paź 2024, 21:54 użytkownik Kris ***@***.***> napisał:

**Comment by frytex2** [comment]
I wonder if I could do anything to help you? do you need some data about
the hardware or os version? or maybe i could trace the traffic from the tv
to the servers? (it is probably ssl-encrypted, though)

the trick is it it worked through a few updates. it only stopped after I
unfortunately deleted the profile

śr., 23 paź 2024, 22:18 użytkownik CourvilleSoftware <
***@***.***> napisał:

**Comment by olivergiangvu** [comment]
I have the same err on Sony Bravia Android TV

**Comment by JSCFORCE** [comment]
Anyone with this issue do you have any form of DNS ad blocking enabled?

If the answer is yes, disable it and try again. I had this issue myself.

**Comment by olivergiangvu** [comment]
I dont have any blocker set up. My DNS settings is 1.1.1.1 for premium and
8.8.8.8 for secondary push through from my router.

**Comment by Flavien06** [comment]
Hello, I have the same connection problem on my devices (chromecast, tablet and Android smartphone

I finally using the Apple connection and they work ! , the other ones loin method giving an error. 
I also disabled my custom dns

**Comment by Aruraka** [comment]
> Anyone with this issue do you have any form of DNS ad blocking enabled?
> 
> If the answer is yes, disable it and try again. I had this issue myself.

Hii i don't have any dns or ad blocking enabled in Android 14 but still I cannot connect to trakt.It's showing http error, it's showing http error and also showing can't authenticate
